bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/evaluation/cache/MemCache.kt

```diff
@@ -0,0 +1,23 @@
+package io.bucketeer.sdk.android.internal.evaluation.cache
+
+internal interface MemCache<in Key : Any, Value : Any> {
```

Comment:
We should rename this to `EvaluationMemCache`, or if it's a generic class then we can move it to `io.bucketeer.sdk.android.internal.cache` package

Reply:
yes, Its generic class. Let's me move it as your suggestion

Reply:
changed https://github.com/bucketeer-io/android-client-sdk/pull/89/commits/9d82b92d319042e062f1c4c1b993feca65873716